Citizen-Science/src/screens/LoginScreen.tsx

```diff
@@ -0,0 +1,125 @@
+
+                {/* Second Blob Cluster */}
+                <Blobs rotationDeg={'0deg'} widthPercentage={20} heightPercentage={10} position={{ top: 6, left: 80 }} />
+                <Blobs rotationDeg={'0deg'} widthPercentage={6} heightPercentage={3} position={{ top: 15, left: 93 }} />
+
+                {/* Logo Name */}
+                <LogoName position={'bottomRight'} color={'grey'}/>
+
+                {/* Back Button */}
+                <BackButton />
+
+                {/*  'Welome Back' title on login */}
+                <ScreenTitle
+                    text={'Welcome \nBack'}
+                    textStyle={'title'}
+                    fontSize={5}
+                    color={'white'}
+                    // Uses responsive library {width, height} through the components file
+                    position={{top: 20, left: 5}}
+                />
+
+
+                {/* Email TextField */}
+                <CustomTextField
+                    size={{width: responsiveWidth(70), height: responsiveHeight(5.5)}}
+                    placeholder="Email"
+                    value={email}
+                    onChangeText={setEmail}
+                    borderColor="#5EA1E9"
+                    borderRadius={10}
+                    position={{ horizontal: responsiveWidth(15), vertical: responsiveHeight(61) }}
+                />
+
+
+                {/* Password TextField */}
+                <CustomTextField
+                    size={{width: responsiveWidth(70), height: responsiveHeight(5.5)}}
+                    placeholder="Password"
+                    value={password}
+                    onChangeText={setPassword}
+                    secureTextEntry={true} // To hide password input
+                    borderColor="#5EA1E9"
+                    borderRadius={10}
+                    position={{ horizontal: responsiveWidth(15), vertical: responsiveHeight(68) }}
+                />
+
+                {/* Login Button */}
+                <CustomButton
+                    size={{width: responsiveWidth(70), height: responsiveHeight(5.5)}}
+                    label="Login"
+                    labelColor="white"
+                    backgroundColor="#5EA1E9"
+                    //TODO: potentially redirect to home page after account creation
+                    onPress={() => handleLoginPress()}
+                    position={{ horizontal: responsiveWidth(15), vertical: responsiveHeight(75) }}
+                />
+
+                {/* Giving Users the option to signup if they are not registered */}
+                <Text style={{fontSize: responsiveFontSize(2), position: 'absolute', top: responsiveHeight(84)}}>
+                    Don't have an account?{' '}
+                    <TouchableOpacity onPress={() => navigation.navigate('Signup')}>
+                        <Text style={{fontSize: responsiveFontSize(2),  color: '#5EA1E9', textDecorationLine: 'underline' }}>
```

Comment:
This seems to need slight adjusting, "Sign Up" is not inline with the "Don't have an account?" line.

Reply:
![image](https://github.com/Woope-CitizenScience/Woope/assets/65632734/9501fe51-2ae9-4add-9141-6af1738fc1f2)